net-misc/bird/bird-2.0.12.ebuild

```diff
@@ -49,6 +49,14 @@ src_configure() {
 	# This export makes compilation and test phases verbose
 	export VERBOSE=1
 
+	# lto must be enabled by default as bird is mono-threaded and use several
+	# optimisations to be fast, at it may very likely be exposed to several
+	# thounsand BGP updates per seconds
+	# Although, we make it possible to desactivate it if wanted
+	if use custom-cflags; then
+		export bird_cv_c_lto=no
+	fi
```

Comment:
I would suggest to move it after `local myargs` and append it to the list this way:
```bash
    use custom-cflags && myargs+=( bird_cv_c_lto=no )
```

Reply:
What’s the difference between those two approaches?

Reply:
`export` will unnecessarily polute environmental variables with `bird_cv_c_lto=no`, wich is only needed in configuration phase. The suggested approach will pass it as an argument to `econf` where it is needed, additionally, use flag effect will be directly visible in `build.log`, which is better for debugging. It will look like this in `build.log`:
```
./configure --prefix=/usr --build=x86_64-pc-linux-gnu --host=x86_64-pc-linux-gnu --mandir=/usr/share/man --infodir=/usr/share/info --datadir=/usr/share --sysconfdir=/etc --localstatedir=/var/lib --datarootdir=/usr/share --docdir=/usr/share/doc/bird-2.0.12 --htmldir=/usr/share/doc/bird-2.0.12/html --libdir=/usr/lib64 --localstatedir=/var --enable-client --disable-debug --disable-libssh bird_cv_c_lto=no
```
Anyway, this pattern is common in `::gentoo` tree :)

Reply:
Makes sens, indeed. I’m not on my computer right now but I’ll updated it this evening or first thing in the morning.

Reply:
Updated and rebased :)